logging/logging.go

```diff
@@ -71,7 +71,7 @@ type BackupError struct {
 	Item   string `json:"item"`
 }
 
-type outFunc func(string) error
+type outFunc func(string)
```

Comment:
According to the tests this is referenced somewhere else, too and needs adjusting:
```
restic/wait.go:74:27: cannot use list.out (type func(string) error) as type logging.outFunc in argument to logging.New
```

Reply:
Oops